draft-ietf-acme-acme.md

```diff
@@ -1663,38 +1656,32 @@ server at the domain such that when a handshake is initiated with the Server
 Name Indication extension set to "\<Z[0:32]\>.\<Z[32:64]\>.acme.invalid", the
 generated test certificate is presented.
 
-The response to the DVSNI challenge provides the validation JWS to the server.
+The response to the DVSNI challenge simply acknowledges that the client is ready
+to fulfill this challenge.
 
 type (required, string):
 : The string "dvsni"
 
-validation (required, string):
-: The JWS object computed with the validation object and the account key
+token (required, string):
+: The "token" value from the authorized key object in the challenge.
 
 ~~~~~~~~~~
 {
-  "type": "dvsni",
-  "validation": {
-    "header": { "alg": "RS256" },
-    "payload": "qzu9...6bjn",
-    "signature": "gfj9XqFv07e1wU66hSLYkiFqYakPSjAu8TsyXRg85nM"
-  }
+  "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA"
 }
 ~~~~~~~~~~
 
 Given a Challenge/Response pair, the ACME server verifies the client's control
 of the domain by verifying that the TLS server was configured appropriately.
 
-1. Verify the validation JWS using the account key for which the challenge
-   was issued.
-2. Decode the payload of the JWS as UTF-8 encoded JSON.
-3. Verify that there are exactly two fields in the decoded object, and that:
-  * The "type" field is set to "dvsni"
-  * The "token" field matches the "token" value in the challenge
-4. Open a TLS connection to the domain name being validated on port 443,
+1. Verify that the "token" value in the response matches the "token" field in
```

Comment:
As discussed on the list, this check helps ensure that an intermediary didn't provide a bogus authorized key object.

Also, please raise these issues on the mailing list rather than using the PR thread.